fast64_internal/sm64/sm64_level_writer.py

```diff
@@ -445,6 +445,29 @@ def replaceSegmentLoad(levelscript, segmentName, command, changedSegment):
     changedLoad[1][2] = segmentName + "SegmentRomEnd"
 
 
+def replaceScriptLoads(levelscript, obj):
+    newFuncs = []
+    for jumpLink in levelscript.levelFunctions:
+        target = jumpLink[1][0]  # format is [macro, list[args], commnent]
```

Comment:
```suggestion
        target = jumpLink[1][0]  # format is [macro, list[args], comment]
```
I have no idea what the sm64 sorcery is so I'm glad I can point out trivial things <img src="https://cdn.frankerfacez.com/emoticon/595482/2" alt="FeelsAmazingMan" height="32">